src/app/shared/utils/short-number.pipe.ts

```diff
@@ -0,0 +1,40 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { isEmpty } from '../empty.util';
+
+
+@Pipe({
+  name: 'dsShortNumber'
+})
+export class ShortNumberPipe implements PipeTransform {
+
+  transform(number: number, args?: any): any {
+    // will only work value is a valid number
+    if (isNaN(number) || isEmpty(number) || number === 0) {
+      return number;
+    }
+
+    let abs = Math.abs(number);
+    const rounder = Math.pow(10, 1);
+    const isNegative = number < 0; // will also work for Negetive numbers
+    let key = '';
+
+    const powers = [
+      {key: 'Q', value: Math.pow(10, 15)},
+      {key: 'T', value: Math.pow(10, 12)},
+      {key: 'B', value: Math.pow(10, 9)},
+      {key: 'M', value: Math.pow(10, 6)},
+      {key: 'K', value: 1000}
```

Comment:
Intuitively I'd expect lower case letters, e.g. `15k` rather than `15K`. However I'm not a native english speaker, and I can find sources online for using either. @tdonohue Do you think capitalization matters in this case?

Reply:
Capitalization doesn't really matter in this case.  I'd just want consistency & this is consistent. So, I'm fine with uppercase letters here.